src/pages/TransactionDetailsPage.vue

```diff
@@ -0,0 +1,276 @@
+const isFetchingTransaction = ref(false);
+const instructionsTable = useTable(() => http.fetchInstructions(transactionHash.value));
+
+watch(
+  () => transactionHash.value,
+  async () => {
+    try {
+      isFetchingTransaction.value = true;
+      const res = await Promise.all([http.fetchTransaction(transactionHash.value), instructionsTable.fetch()]);
+      transaction.value = res[0];
+    } catch (e) {
+      handleUnknownError(e);
+    } finally {
+      isFetchingTransaction.value = false;
+    }
+  },
+  { immediate: true }
+);
+</script>
+
+<template>
+  <div class="transaction-details">
+    <BaseContentBlock
+      class="transaction-details__metrics"
+      :title="$t('transactions.transactionDetails')"
+    >
+      <template #default>
+        <div
+          v-if="isFetchingTransaction"
+          class="transaction-details__info_loading"
+        >
+          <BaseLoading />
+        </div>
+        <div v-else-if="transaction">
+          <div class="transaction-details__info">
+            <div class="transaction-details__info-row">
+              <DataField
+                :title="$t('transactions.transactionHash')"
+                :hash="transaction.hash"
+                :type="transactionHashType"
+                copy
+              />
+
+              <div class="transaction-details__info-row-time">
+                <span class="h-sm">{{ $t('transactions.timestamp') }}</span>
+                <div class="transaction-details__info-row-time-date row-text">
+                  <TimeIcon />
+                  <span>{{ getUTCFormat(transaction.created_at) }}</span>
+                </div>
+              </div>
+            </div>
+            <div class="transaction-details__info-row">
+              <div class="transaction-details__info-row-status">
+                <span class="h-sm">{{ $t('transactions.status') }}</span>
+                <TransactionStatus
+                  :committed="transaction.status === 'Committed'"
+                  type="label"
+                />
+              </div>
+
+              <DataField
+                v-if="transaction.rejection_reason"
+                :title="$t('transactions.rejectedReason')"
+                :value="transaction.rejection_reason"
+              />
+            </div>
+            <div class="transaction-details__info-row">
+              <DataField
+                :title="$t('accounts.accountId')"
+                :hash="transaction.authority"
+                :type="hashType"
+                :link="`/accounts/${transaction.authority}`"
+                copy
+              />
+
+              <DataField
+                :title="$t('transactions.nonce')"
+                :value="transaction.nonce"
+              />
+            </div>
+            <div class="transaction-details__info-row">
+              <DataField
+                :title="$t('transactions.block')"
+                :hash="String(transaction.block)"
```

Comment:
Why this prop is called `:hash`? It doesn't make any sense here.

Reply:
With `:hash` prop it uses `BaseHash` component otherwise it uses `span`. Probably I can make it a boolean. Should I?

Reply:
Why `<BaseHash>` for `transaction.block`?